### PR TITLE
Cursor pagination

### PR DESCRIPTION
**Pull request by Ptr314**
First version to test it wider

### CODE REVIEW

File: zenpy/lib/response.py [hunk 1]
```diff
@@ -116,10 +116,17 @@ def build(self, response):
         # Collection of objects (eg, users/tickets)
         plural_object_type = as_plural(self.api.object_type)
         if plural_object_type in zenpy_objects:
-            return ZendeskResultGenerator(
-                self,
-                response_json,
-                response_objects=zenpy_objects[plural_object_type])
+            meta = response_json.get('meta')
```
Comment (cryptomail) [inline zenpy/lib/response.py:119]: 👨‍🍳 💋 good stuff, this is in fact a great sentinel

**Comment by kenny291** [comment]
@Ptr314 , I updated the script to support CBP for API: user, satisfaction_ratings.

**Comment by Ptr314** [comment]
Hi @kenny291 !

Thank you for your contributions!

Zendesk has finally decided to move here: https://github.com/bersama-sama/zenpycbp so could you be so kind to repeat your changes there?

**Comment by Ptr314** [comment]
Hi @kenny291 

Managed to move your commits to a new repo: https://github.com/bersama-sama/zenpycbp/pull/5 

As they are made when a new version 2.0.27 is already published and has previous commits merged, I'll need to check and find a better way to incorporate them.

**Comment by kenny291** [comment]
Thank you @Ptr314

**Pull request closed without merging**